taxi/views.py

```diff
@@ -1,3 +1,20 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from .models import Driver, Manufacturer, Car
```

Comment:
add blank line between different import groups
use absolute import instead of relative